bricksrc/equipment.py

```diff
@@ -439,6 +439,28 @@
     "AHU": {
         "tags": [TAG.Equipment, TAG.AHU],
         "subclasses": {
+            "Dedicated_Outdoor_Air_System": {
+                "tags": [
+                    TAG.Equipment,
+                    TAG.Dedicated,
+                    TAG.Outdoor,
+                    TAG.Air,
+                    TAG.System,
+                ],
+                OWL.equivalentClass: BRICK["DOAS"],
+            },
+            "DOAS": {
```

Comment:
This could still be `DOAS` if that's a common abbreviation

Reply:
Yeah, that is the common abbreviation.